modules/serverless-cost-management-labels.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * /serverless/integrations/serverless-cost-management-integration.adoc
+
+:_content-type: REFERENCE
+[id="serverless-cost-management-labels_{context}"]
+= Using labels for cost management queries
+
+Labels, which are referred to in cost management as _tags_, can be applied for nodes, namespaces or pods. Each label is a key and value pair. You can use a combination of multiple labels to generate reports. You can access reports about costs by using the link:https://console.redhat.com/openshift/cost-management/[Red Hat hybrid console].
```

Comment:
`which are referred to in cost management as _tags_` is a little bit of a mouthful. Could this be condensed, perhaps `also known as _tags_ in cost management` ?

Reply:
Done, good suggestion, thanks!